src/method_ilan.jl

```diff
@@ -112,16 +114,16 @@ function ilan(
     a=Vector{T}(γ.^(0:2m+2)); a[1]=zero(T);
     local M0inv::LinSolver = create_linsolver(linsolvercreator,nep,σ)
     err=ones(m,m);
```

Comment:
This does not seem like a good way to allocate memory for error. Type is not set (should probably be `real(T)`). It is better to initiate it to NaN's.

Reply:
The problem with output incorrect can be ignored (sorry). It seems to just be that the inner solver only returns one solution. This call works: `λ,v=ilan(nep;v=v0,tol=1e-5,neigs=3,maxit=50,logger=1,check_error_every=1,inner_solver_method=IARInnerSolver(maxit=50));`